source/agora/node/Ledger.d

```diff
@@ -31,6 +33,7 @@ public class Ledger
     /// Ctor
     public this ()
     {
+        this.ledger.reserve(2 ^^ 20);  // reserve ~1 million blocks
```

Comment:
Yeeeeeaaaaaaaaaahhhhhhhhhhhh... About that... I have mixed feelings about allocating 151Mbs like this with the GC, even if it's just a reserve.